packages/grafana-data/src/text/markdown.ts

```diff
@@ -42,3 +42,15 @@ export function renderTextPanelMarkdown(str?: string, options?: RenderMarkdownOp
 
   return sanitizeTextPanelContent(html);
 }
+
+export function renderChatMarkdown(str?: string): string {
```

Comment:
necessary?   do we want/need something different than what we are doing in the text panel?